CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ project(
 
 # [CMAKE.SKIP_EXAMPLES]
 option(
-    BEMAN_EXEMPLAR_BUILD_EXAMPLES
+    BEMAN_INPLACE_VECTOR_BUILD_EXAMPLES
     "Enable building examples. Default: ON. Values: { ON, OFF }."
```

Comment:
Doc string should be changed to "Enable building examples. Default: ` ${PROJECT_IS_TOP_LEVEL}`", so the doc string is correct when the project is consumed via `add_subdirectory`.

Please fix the other `options()` too while we're at it.